buildSrc/src/main/java/Dependencies.kt

```diff
@@ -4,7 +4,7 @@ object Dependencies {
         val okhttp = "3.14.6"
         val sentry = "1.7.23"
         val retrofit = "2.6.2"
-        val kotlin = "1.3.61"
+        val kotlin = "1.3.70"
```

Comment:
Is our goal to delete this?

Reply:
we should try, kotlin plugin supports provides "platform" data